pkg/controller/config/config_controller.go

```diff
@@ -153,7 +156,6 @@ func (r *ReconcileConfig) Reconcile(request reconcile.Request) (reconcile.Result
 	}
 
 	newSyncOnly := newSet()
-	toClean := newSet()
 	if instance.GetDeletionTimestamp().IsZero() {
 		if !hasFinalizer(instance) {
 			instance.SetFinalizers(append(instance.GetFinalizers(), finalizerName))
```

Comment:
We can remove this finalizer too. Let's leave that for a follow-on PR in the interest of incremental changes.